resources/views/cd-index.blade.php

```diff
@@ -1,4 +1,4 @@
-<x-cwd-framework.layout.app title=":project_name" subtitle="Custom Development">
+<x-cd.layout.app title="Laraveltest" subtitle="Custom Development">
```

Comment:
Not sure how this happened, but the title should be left as a token:
```suggestion
<x-cd.layout.app title=":project_name" subtitle="Custom Development">
```

Reply:
I remember being prompted for the project name at one point.